DataTransferObject/UserDataTransferObject.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\DataTransferObject;
+
+use SumoCoders\FrameworkMultiUserBundle\User\User;
+
+interface UserDataTransferObject
+{
+    public static function fromUser(User $user);
+    public function getEntity();
```

Comment:
Doesn't a factory process it's data immediatly? I see these objects more as a temporary storage of data